### PR TITLE
2024 ESDS Annual Event Recap

### PR DESCRIPTION
**Pull request by kafitzgerald**

### CODE REVIEW

File: posts/2024/esds-annual-event-recap.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Tropical Cyclone Track Extraction from Unstructured Grids - Dave Ahijevych
+
+## Takeaways
+
+## Acknowledgements
+Thanks to the following individuals who helped make this event happen!
+
+Events Working Group: Teagan King, John Clyne, Orhan Eroglu,...
+Tutorials: Julia Kent, Anissa Zacharias, Philip Chmielowiec, Stas Jaroszynski,...
```
Comment (TeaganKing) [inline posts/2024/esds-annual-event-recap.md:59]: Julia Kent, Cora Schneck, Katelyn FitzGerald, Mea Trahan, Negin Sobhani, Brian Vanderwende, Orhan Eroglu, Philip Chmielowiec, Scott Pearse, Nihanth Cherukuru, Stas Jaroszynski

File: posts/2024/esds-annual-event-recap.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Collaborative Work Time Hosts: ...
```
Comment (TeaganKing) [inline posts/2024/esds-annual-event-recap.md:60]: Mike Levy, Teagan King,   Dan Adriaansen, Anissa Zacharias, Katelyn FitzGerald, Nick Cote, Dave Ahijevych

File: posts/2024/esds-annual-event-recap.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Xarray - Negin Sobhani ([recording](), [slides]())
+
+Scaling from Single Cored to HPC with Dask - Brian Vanderwende ([recording](), [slides]())
+
+Interactive Unstructured Grids Visualization with UXarray - Orhan Eroglu, Philip Chmielowiec, and Rajeev Jain ([recording](), [slides]())
+
+3D Visualization with VAPOR - Scott Pearse, Stas Jaroszynski, and Nihanth Cherukuru ([recording](), [slides]())
+
+### Collaborative Work Time
+CUPiD - Teagan Kind and Mike Levy
```
Comment (TeaganKing) [inline posts/2024/esds-annual-event-recap.md:43]: Minor typo: "King"

File: posts/2024/esds-annual-event-recap.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+This blog post shares a brief overview of the event followed by resources and takeaways
+from the event.
+
+## Overview
+
+The two day event consisted of a series of tutorials, office hours, and collaborative
+work sessions as well as a talk from our invited speaker to kick off the event. You
+can find the [full agenda here](https://docs.google.com/spreadsheets/d/1XNZW8NWoF2LaQ4mlo8c06lNnNVJ0ndTG_DfFH8KsatQ/view).
+
+Something about the purpose of the event...
```
Comment (TeaganKing) [inline posts/2024/esds-annual-event-recap.md:22]: - Provide access to introductory and advanced Earth system data science focused tutorials
- Encourage collaboration and open development
- Build a welcoming and inclusive community

File: posts/2024/esds-annual-event-recap.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+## Resources
+
```
Comment (TeaganKing) [inline posts/2024/esds-annual-event-recap.md:25]: I suppose we could share the youtube channel here, information from [this slide deck](https://docs.google.com/presentation/d/1S298Owkmf-1XIW9qdtmNmBNzdzx4eT_mRAl8Qj6AzOs/edit#slide=id.g265ff7beb14_0_0), and/or info from the google drive from the event if we have anything we'd like to share there?

**Comment by kafitzgerald** [comment]
Thanks for the input Teagan!  Should all be integrated now and tidied up.

**Review by TeaganKing — APPROVED** [review]
Looks great! Thanks Katelyn!

**Pull request merged**